pkg/indexers/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- ncdc
+- sttts
+- shawn-hurley
```

Comment:
I would probably not think that I have enough knowledge here.

Reply:
removed